content/en/docs/concepts/traffic-management/index.md

```diff
@@ -254,502 +172,246 @@ spec:
+lets users send traffic to two separate services, ratings and reviews, as if
+they were part of a bigger virtual service at `http://bookinfo.com/.` The
+virtual service rules match traffic based on request URIs and direct requests to
+the appropriate service.
 
 {{< text yaml >}}
 apiVersion: networking.istio.io/v1alpha3
 kind: VirtualService
 metadata:
-  name: productpage
+  name: bookinfo
 spec:
   hosts:
-  - productpage
+    - bookinfo.com
   http:
   - match:
     - uri:
-        prefix: /api/v1
+        prefix: /reviews
     route:
-    ...
-{{< /text >}}
-
-#### Multiple match conditions {#multi-match}
-
-Conditions can have multiple matches simultaneously. In such cases, you use the
-nesting of the conditions in the routing rule to specify whether AND or OR
-semantics apply. To specify AND semantics, you nest multiple conditions in a
-single section of `match.`
-
-For example, the following rule applies only to requests for a
-URL that begins with `/api/v1` **and** that include the custom
-`end-user` header that contains the exact value `jason`:
-
-{{< text yaml >}}
-apiVersion: networking.istio.io/v1alpha3
-kind: VirtualService
-metadata:
-  name: productpage
-spec:
-  hosts:
-  - productpage
-  http:
+    - destination:
+        host: reviews
   - match:
     - uri:
-        prefix: /api/v1
-      headers:
-        end-user:
-          exact: jason
+        prefix: /ratings
     route:
-    ...
-{{< /text >}}
-
-To specify OR conditions, you place multiple conditions in separate sections of
-`match.` Only one of the conditions applies. For example, the following rule
-applies to requests for a URL that begins with `/api/v1` **or** to requests with the
-custom `end-user` header containing the exact value `jason`:
+    - destination:
+        host: ratings
+...
 
-{{< text yaml >}}
-apiVersion: networking.istio.io/v1alpha3
-kind: VirtualService
-metadata:
-  name: productpage
-spec:
-  hosts:
-  - productpage
   http:
   - match:
-    - uri:
-        prefix: /api/v1
-    - headers:
-        end-user:
-          exact: jason
+      sourceLabels:
+        app: reviews
     route:
-    ...
+...
 {{< /text >}}
 
-{{< warning >}}
-In the two examples, above, the only difference between AND behavior and OR behavior is
-an extra `-` character in front of the `headers` field. The `-` in the YAML representation,
-indicates two separate matches as opposed to one match with multiple conditions.
-{{< /warning >}}
-
-### Routing rule precedence {#precedence}
+For some match conditions, you can also choose to select them using the exact
+value, a prefix, or a regex.
 
-Multiple rules for a given destination in a configuration file are evaluated in
-the order they appear. The first rule on the list has the highest priority.
+You can add multiple match conditions to the same `match` block to AND your
+conditions, or add multiple match blocks to the same rule to OR your conditions.
```

Comment:
Hmm. We used to have AND and OR match examples here, which people found helpful because I believe they are the only such examples we have anywhere in the docs.

Reply:
Hmm indeed. I was trying to cut down the number of examples because this is meant to be an introductory concepts doc, not a config reference (if your concepts docs/tutorials have become a reference it's generally a sign that content needs to be moved....). I thought everything was covered somewhere else but apparently not. Is there another good place to put the AND and OR examples?

Reply:
I guess they should go into the ref docs. Even without examples, with your text in the concepts that says AND and OR is possible, people will hopefully be able to figure it out. Let's go with that assumption for now.

Reply:
OK. Should we add an issue to add them to the refdocs?

Reply:
Let's wait to see if they're really needed.